pkgs/by-name/bi/bitrise/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+buildGoModule rec {
+  pname = "bitrise";
+  version = "2.19.0";
+
+  src = fetchFromGitHub {
+    owner = "bitrise-io";
+    repo = "bitrise";
+    rev = version;
+    hash = "sha256-VjuDeRl/rqA7bdhn9REdxdjRon5WxHkFIveOYNpQqa8=";
+  };
+
+  doCheck = false; # many tests rely on writing to $HOME/.bitrise
```

Comment:
You could do something like:
```
preCheck = "export HOME=`mktemp -d`";
```

Reply:
Thank you, I didn't think of this trick. I'd prefer merging this first though, then I'll take another stab at running the tests later if that's okay for you.

Reply:
I started working around the failing tests, but after patching $HOME, I found out that some tests rely on network access, so this is going be a longer journey